processing/data_collection/gazette/spiders/rn_mossoro.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime, date
+
+import dateutil
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMossoroSpider(BaseGazetteSpider):
+
+    name = "rn_mossoro"
+    allowed_domains = ["jom.prefeiturademossoro.com.br"]
+
+    TERRITORY_ID = "2408003"
+
+    def __init__(self, start_date=None, end_date=None, *args, **kwargs):
+        self.start_date = date.today().replace(day=1)
+        self.end_date = date.today()
+
+        super(RnMossoroSpider, self).__init__(start_date, end_date, *args, **kwargs)
+
+        self.logger.debug(
+            "Start date is {date}".format(date=self.start_date.isoformat())
+        )
+        self.logger.debug("End date is {date}".format(date=self.end_date.isoformat()))
+
+    def start_requests(self):
+
+        for year in range(self.start_date.year, self.end_date.year + 1):
+            for month in range(self.start_date.month, self.end_date.month + 1):
+                base_url = f"http://jom.prefeiturademossoro.com.br/{year}/{month}/"
+                yield scrapy.Request(base_url)
+
+    def parse(self, response):
+        for entry in response.css("article.post.category-jom"):
+
+            url = entry.css("a:first-of-type::attr(href)").get()
+            datetime_meta = entry.css(
+                "div.post-data-text time.published::attr(datetime)"
```

Comment:
`div.post-data-text` also doesn't seem necessary at a first glance and we could simplify this selector. What do you think?

Reply:
Again correct, just being more explicit and specific.

Reply:
Again, not too important, so I'll just wait for your closing thoughts.

Comparing to the last one, this introduces even more complexity. The time-related stuff seems to be enough.

Reply:
Right. Changed.